src/components/DisplayComments.jsx

```diff
@@ -1,36 +1,72 @@
 import { useEffect, useState } from "react";
 import { Link, useParams } from "react-router-dom";
-import * as api from '../utils/api'
+import * as api from '../utils/api';
+import moment from "moment";
 
 function DisplayComments() {
     const { article_id } = useParams();
-    const [comments, setComments] = useState([])
+    const [comments, setComments] = useState([]);
+    const [users, setUsers] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    const handdleSubmit = (e) => {
+        e.preventDefault();
+    }
     
     useEffect(() => {
         async function fetchComments(articleId) {
-            let request = await api.getCommentsByArticleId(articleId)
-            console.log(request)
-            setComments(request)
+            let request = await api.getCommentsByArticleId(articleId);
+            setIsLoading(false);
+            setComments(request);
+        }
+
+        async function fetchUsers() {
+            let request = await api.getUsers()
+            setUsers(request);
         }
 
-        fetchComments(article_id)
+        fetchComments(article_id);
+        fetchUsers();
     }, [article_id])
 
+    if (isLoading) {
+        return (
+            <div id="article_container">
+                <section id="article_section">
+                <p>Loading...</p><div className="loader"></div>
+                </section>
+            </div>
+        )
+    }
+
     return (
         <div id='comment_container'>
             <Link to={`/articles/${article_id}`} id='return_link'>⥄ back</Link>
-            <h2>COMMENTS</h2>
+            <h2 className='comment_title'>COMMENTS</h2>
 
             <ul id="comment_section">
                 {comments.map(comment => {
-                    return (
-                        <li key={comment.comment_id}>
-                            <article id="comment">
-                                <h3>{comment.author}</h3>
-                                <p>{comment.body}</p>
-                            </article>
-                        </li>
-                    )
+                    return users.map(user => {
+                        if(user.username === comment.author) {

```

Comment:
Can you account for if an article has no comments?
Something like 'no comments to display'